README.txt

```diff
@@ -602,6 +603,12 @@ VMA_TCP_3T_RULES
 Use only 3 tuple rules for TCP, instead of using 5 tuple rules.
 This can improve performance for a server with listen socket which accept many
 connections.
+Default: 0 (Disable)
+
+VMA_UDP_3T_RULES
+Use only 3 tuple rules for UDP.
+This can reduce hardware resource to manage flow steering rules.
+Default: 1 (Enable)
```

Comment:
This parameter can be relevant in case application uses connected udp sockets. 3 tuple rules are used in hardware flow steering rule when it is enabled and 5 tuple flow steering rule when it is disabled.
Enabling this option can reduce hardware flow steering resources.
But when it is disabled application might see benefits in latency and cycles per packet.